src/_tests_/SeachResults.test.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SearchResults from "../components/SearchResults";
+
+describe("SearchResults", () => {
+  it("renders correctly", () => {
+    const results = [];
+    const { asFragment } = render(<SearchResults results={results} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("renders the correct text", () => {
+    const results = [];
+    render(<SearchResults results={results} />);
+    expect(screen.getByText("No results!")).toBeInTheDocument();
+  });
+});
```

Comment:
You could put the results variable outside of both tests to save having to declare it twice:
```
describe("SearchResults", () => {
  const results = [];
  it("renders correctly", () => {
    const { asFragment } = render(<SearchResults results={results} />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("renders the correct text", () => {
    render(<SearchResults results={results} />);
    expect(screen.getByText("No results!")).toBeInTheDocument();
  });
});
```